compiler/src/main/java/org/qbicc/graph/MemoryAtomicityMode.java

```diff
@@ -47,6 +60,15 @@ public static MemoryAtomicityMode min(MemoryAtomicityMode a, MemoryAtomicityMode
         return a.ordinal() < b.ordinal() ? a : b;
     }
 
+    /**
+     * Get the closest corresponding access mode, which may have to be transformed further before use.
+     *
+     * @return the access mode (not {@code null})
+     */
+    public AccessMode getAccessMode() {
+        return  accessMode;
+    }
```

Comment:
This method should hopefully allow us to more easily bridge over to the new API.